### ISSUE TITLE
Async Pointing

### ISSUE THREAD

**Issue by alexcouper** [pretenders/ployst#55]
It'd be a great feature going forward to enable asnyc pointing from within ployst.

This was suggested by @koddsson.

Members would assign a secret point to an issue and once everyone has given their vote all is revealed (and placed on the issue itself). If there's a discrepancy people can now chat on the github issue itself and then repoint.

**Comment by alexcouper** [comment]
An example of a request for this feature elsewhere:
http://support.scrumdo.com/discussions/questions/1096-planning-poker-for-distributed-teams

**Comment by txels** [comment]
Yes I'd like to see ployst growing as a tool to also curate/layer additional information on top of stories/features (or maybe even potentially being the first source of story data). This looks like one nice such addition.

Maybe even better, this could be done by a separate tool that is integrated as a ployst provider.

**Comment by alexcouper** [comment]
A while ago we spoke about the idea that providers would themselves own no data.

If we are to keep that constraint and add things like this, I believe we will need to provide another sort of "dumping ground" (like settings) for providers centrally. 

Either that or remove this constraint.

**Comment by txels** [comment]
I'm not sure I understand 100% what you mean but e.g. a planning provider actually creates and pushes story data into ployst core, a version control provides pushes branch data etc... So giving providers the ability to attach additional data to a story does not feel like deviating much from what we already have, and we need to provide a clean mechanism for that (give the "dumping ground" some structure, if I get what you mean correctly, as in define a proper model for additional "feature" metadata - or additional metadata for any of the core models we have [repo, feature, branch...]).

I think this is better than keeping data in providers, because if done cleanly other providers may benefit from this 3rd party data, and the UI can present that data in a unified way regardless of origin.
